core/auth/module.go

```diff
@@ -0,0 +1,33 @@
+package auth
+
+import (
+	"flamingo.me/dingo"
+	interfaces2 "flamingo.me/flamingo/v3/core/auth/interfaces"
```

Comment:
AFAIS there's no need to use the alias.

```suggestion
	"flamingo.me/flamingo/v3/core/auth/interfaces"
```